.github/workflows/rust.yml

```diff
@@ -84,9 +84,13 @@ jobs:
     name: Nix-Build Tests
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: cachix/install-nix-action@v15
+      - uses: actions/checkout@v2.4.0
+      - uses: cachix/install-nix-action@v16
+        with:
+          nix_path: nixpkgs=channel:nixos-unstable
```

Comment:
Did you try running it on stable? Would be preferable imo (most production systems will use that), or even better run both (although that could become annoying on breaking changes).